dotcom-rendering/src/web/components/Callout/Select.tsx

```diff
@@ -1,36 +1,37 @@
-import { CampaignFieldSelect } from '../../../types/content';
-import { FieldLabel } from './FieldLabel';
+import { Select as SourceSelect } from '@guardian/source-react-components';
+import type { CampaignFieldSelect } from 'src/types/content';
 
 type Props = {
+	validationErrors?: { [key in string]: string };
 	formField: CampaignFieldSelect;
 	formData: { [key in string]: any };
 	setFormData: React.Dispatch<React.SetStateAction<{ [x: string]: any }>>;
 };
 
-export const Select = ({ formField, formData, setFormData }: Props) => (
-	<>
-		<FieldLabel formField={formField} />
```

Comment:
Now that we're using the `Select` component from Source, there is no need to use a custom `FieldLabel` - once again thanks for pointing that out @bryophyta